src/main/java/chad/task/Todo.java

```diff
@@ -0,0 +1,14 @@
+package chad.task;
+
+/**
+ * chad.task.Todo task that is child of task
+ */
+public class Todo extends Task {
+    public Todo(String description) {
+        super(description);
+    }

```

Comment:
Perhaps you can include a Javadoc comment for this constructor? So that users do not need to read the code to understand if the constructor does anything else.